tools/Plotting the shear profile of the E06 floating lidar data utilising the WRA Data Model and the brightwind library.ipynb

```diff
@@ -0,0 +1,415 @@
+   "source": [
+    "The Name and Lat, Long values can be seen in the top half of the table. The bottom half contains the two vertical profiler\n",
+    "properties used in this measurement campaign. One is for the lidar itself and the other is for an ADCP which is mounted underneath the buoy pointing down towards the sea floor.\n",
+    "\n",
+    "Note the `Device Datum Plane Height [m]` for the lidar is stated as 1.6 m above mean sea level.\n",
+    "\n",
+    "### Show all the logger configurations that exist for this station"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5a0eeb75",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meas_sta.logger_main_configs.get_table()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16c71472",
+   "metadata": {},
+   "source": [
+    "### Show all the measurements been made by this station"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "140f2275",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meas_sta.measurements.get_table()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f3bbfb5",
+   "metadata": {},
+   "source": [
+    "There are far to many rows for the dataframe to display so we can filter for just the horizontal wind speeds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "374c7a50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meas_sta.measurements.get_table(wind_speeds=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f6e19ae",
+   "metadata": {},
+   "source": [
+    "You can see all the lidar measurements start with the highest and working down in elevation. The last one is the onboard met station that is mounted on the buoy itself.\n",
+    "\n",
+    "## Load the timeseries data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6086ad9c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = bw.load_csv('E06_Hudson_South_10_min_avg_20190904_20220327.csv', engine='python').apply(pd.to_numeric, \n",
+    "                                                                                               errors='coerce')\n",
+    "data.columns = data.columns.str.strip()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3663f97",
+   "metadata": {},
+   "source": [
+    "Normally loading a csv into the brightwind library would be as simple as `bw.load_csv('filename.csv')`. However, due to poor formatting of the timeseries data file downloaded directly from the above link there are few extra things to do to read this in properly.\n",
+    "1. `engine='python'` is added due to the 3 leading character spaces before the 'NaN' values, Pandas throws an error as it thinks there are multiple data types, strings and numeric in the one column.\n",
+    "1. Also, due to this we need to force this to a numeric by using `.apply(pd.to_numeric, errors='coerce')`\n",
```

Comment:
This also forces all string columns to be NaN as they can't be converted. This only affects the last column, FLS200_S/N, which is the buoy's S/N. Maybe add a comment here commenting that, as the next cell shows all columns and displays NaNs for FLS200_S/N.

Reply:
Thanks for pointing out. I missed that.
Done.